products/protractor.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+changelogTemplate: https://github.com/angular/protractor/blob/master/CHANGELOG.md
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
+
+auto:
+-   npm: protractor
+
+identifiers:
+-   purl: pkg:npm/protractor
+
+releases:
+-   releaseCycle: "7.0"
+    support: 2023-08-31
+    eol: true
+    extendedSupport: true
+    latest: "7.0.0"
+    latestReleaseDate: 2020-05-13
+    releaseDate: 2020-05-13
+    link: https://github.com/angular/protractor/commit/71771dee4e87526dffb725cacac3f9f9e1efc287?short_path=06572a9#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed
```

Comment:
This is confusing because it highlights some differences that I don't know what I'm looking at.

I suggest replacing the link to point to the `CHANGELOG.md` and use anchors for the respective release